### PR TITLE
Add test

### PR DESCRIPTION
**Pull request by Korbin-Wang**
Please~

### CODE REVIEW

File: src/MDSearchBox.jsx [hunk 1]
```diff
@@ -49,8 +49,8 @@ MDSearchBox.propTypes = {
 
 MDSearchBox.defaultProps = {
   text: '',
-  width: 300,
-  height: 42,
+  // width: 300,
```
Comment (jimexist) [inline src/MDSearchBox.jsx]: commented out code should be removed

File: test/MDSearchBox.spec.js [hunk 2]
```diff
@@ -8,7 +8,81 @@ import MDSearchBox from 'src/MDSearchBox';
...
+    before(() => {
+      wrapper = shallow(
+        <MDSearchBox
+          width={500}
+          height={300}
+        /> );
+    });
+
+    it('size should change', () => {
+      // console.log(wrapper.props().style);
```
Comment (jimexist) [inline test/MDSearchBox.spec.js]: commented out code should be removed

File: lib/MDSearchBox.css [hunk 3]
```diff
@@ -9,7 +9,7 @@
       -ms-flex-align: center;
           align-items: center;
 
-  width: 300px;
+  /*width: 300px;*/
```
Comment (ZhengHe-MD) [inline lib/MDSearchBox.css:12]: 如果default props里面有width，那么这里是可以把width删除的。如果你把default props里面的width删了，那么这里就不能删。我觉得前者比较合理和直观

File: lib/MDSearchBox.js [hunk 4]
```diff
@@ -85,9 +85,7 @@ MDSearchBox.propTypes = {
 };
 
 MDSearchBox.defaultProps = {
-  text: '',
-  width: 300,
-  height: 42
```
Comment (ZhengHe-MD) [inline lib/MDSearchBox.js:94]: better put it back.

File: src/MDSearchBox.css [hunk 5]
```diff
@@ -3,7 +3,7 @@
   justify-content: space-between;
   align-items: center;
 
-  width: 300px;
+  /*width: 300px;*/
```
Comment (ZhengHe-MD) [inline src/MDSearchBox.css:6]: same. You can remove it given that the with in defaulProps are still there

Comment (ZhengHe-MD) [inline src/MDSearchBox.css:6]: same. You can remove it given that the width in defaulProps are still there

File: test/MDSearchBox.spec.js [hunk 6]
```diff
@@ -8,7 +8,81 @@ import MDSearchBox from 'src/MDSearchBox';
...
+    let wrapper;
 
+    before(() => {
+      wrapper = shallow(<MDSearchBox />);
+    });
+
+    it('should have no input value', () => {
+        wrapper
+        .childAt(0)
+        .childAt(1)
```
Comment (ZhengHe-MD) [inline test/MDSearchBox.spec.js]: .props()
.value

File: test/MDSearchBox.spec.js [hunk 7]
```diff
@@ -8,7 +8,81 @@ import MDSearchBox from 'src/MDSearchBox';
...
+      wrapper = shallow(
+        <MDSearchBox
+          width={500}
+          height={300}
+        /> );
+    });
+
+    it('size should change', () => {
+      // console.log(wrapper.props().style);
+      expect(wrapper.props().style).toEqual({width:500, height:300});
```
Comment (ZhengHe-MD) [inline test/MDSearchBox.spec.js:76]: better split that assertion into two assertions.
expect(wrapper.props().style.width).toEqual(500);
expect(wrapper.props().style.height).toEqual(300);

or you can use `include`
expect(wrapper.props().style).to.include({ width:500, height:300 })

**Pull request merged**